src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/model/caserecord/ExceptionRecord.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.model.caserecord;
+
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.model.CaseRecord;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Envelope;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDataContent;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.client.model.Classification;
+
+public class ExceptionRecord implements CaseRecord {
+
+    private final Envelope envelope;
+    private final CaseDetails caseDetails;
+
+    public ExceptionRecord(Envelope envelope, CaseDetails caseDetails) {
+        this.envelope = envelope;
+        this.caseDetails = caseDetails;
+    }
+
+    @Override
+    public CaseDataContent createDataContent(
+        Classification classification,
+        String eventToken,
+        boolean ignoreWarning
+    ) {
+        return null;
```

Comment:
Hmm should there be an exception rather than null ... or a Null object ... 
I.e. a CaseDataContent that has nothing ... 
Or an optional ? 

Or have an interface that does not have get CaseRecord and compose the interfaces ....
EnvelopeContainer
CaseDetailsContainer
CaseDataContainer 
and have a visitor to to retrieve the info or use a instance of ?

❌

Reply:
Should have thrown exception in the first place. Will add